tango/__init__.py

```diff
@@ -15,7 +15,7 @@
     "Workspace",
 ]
 
-from tango.executor import Executor
+from tango.executors import Executor
```

Comment:
For consistency with `Step`, `Workspace`, `StepCache`, etc., I think we should keep the base class in `tango/executor.py`, and then put subclasses in `tango/executors/*`.